app/api/server/api.js

```diff
@@ -241,7 +241,7 @@ class APIClass extends Restivus {
 					try {
 						const shouldVerifyRateLimit = rateLimiterDictionary.hasOwnProperty(objectForRateLimitMatch.route)
 							&& (!this.userId || !hasPermission(this.userId, 'api-bypass-rate-limit'))
-							&& (settings.get('API_Enable_Rate_Limiter_Dev') || (process.env.NODE_ENV === 'development' && settings.get('API_Enable_Rate_Limiter_Dev') === true));
+							&& (settings.get('API_Enable_Rate_Limiter') || (process.env.NODE_ENV === 'development' && settings.get('API_Enable_Rate_Limiter_Dev') === true));
```

Comment:
`settings.get('API_Enable_Rate_Limiter') ` should happen before `hasPermission` to prevent load when disabled.

The current case will evaluate `API_Enable_Rate_Limiter_Dev` only if `API_Enable_Rate_Limiter` is disabled what is wrong. The correct should be:

if `API_Enable_Rate_Limiter` enabled test `API_Enable_Rate_Limiter_Dev` and `NODE_ENV` (and the rest)
If `API_Enable_Rate_Limiter` disabled never proceed